app/src/main/java/es/jvbabi/vplanplus/data/repository/BaseDataRepositoryImpl.kt

```diff
@@ -62,35 +63,28 @@ class BaseDataRepositoryImpl(
         }
     }
 
+    @OptIn(ExperimentalEncodingApi::class)
```

Comment:
Maybe take a look at what's causing the need for this annotation

Reply:
Related to https://github.com/VPlanPlus-Project/VPlanPlus/pull/261#discussion_r1491464497

Reply:
3d966c3b9d703c2bba8a8be6b8e904d586d0e6d5